src/status_im/ui/screens/chat/extensions/views.cljs

```diff
@@ -37,8 +37,22 @@
                                   :height    panel-height
                                   :transform [{:translateY bottom-anim-value}]
                                   :opacity   alpha-value}}
-     [react/touchable-highlight
-      {:on-press #(re-frame/dispatch [:wallet/prepare-transaction-from-chat])}
-      [react/view {:width 127 :height 127 :align-items :center :justify-content :center
-                   :background-color (colors/alpha "#887AF9" 0.2) :border-radius 16 :margin-left 8}
-       [react/text "Send transaction"]]]]))
\ No newline at end of file
+     [react/view
+      [react/touchable-highlight
+       {:on-press #(re-frame/dispatch [:wallet/prepare-transaction-from-chat])}
+       [react/view {:width 127 :height 127 :justify-content :space-between
```

Comment:
and here width and height a `128`? those are nicer numbers, sorry for the trouble 🙏